### PR TITLE
Add mention of extensive contract review fee.

### PR DESCRIPTION
**Pull request by josh-padnick**
Here's how this renders on the [pricing page](https://gruntwork.io/pricing/):

image

### CODE REVIEW

File: pages/pricing/_other-services.html [hunk 1]
```diff
@@ -13,5 +13,12 @@ <h2>Other Services</h2>
         </tr>
       </tbody>
     </table>
+
+    <h3>Miscellaneous Fees</h3>
```
Comment (brikis98) [inline pages/pricing/_other-services.html]: `h2`

File: pages/pricing/_other-services.html [hunk 2]
```diff
@@ -13,5 +13,12 @@ <h2>Other Services</h2>
...
+    <p><strong>Gruntwork has no hidden fees.</strong> The one fee we apply is a one-time charge of $4,000 for any significant requested
+    changes on our contract to cover the cost we pay to our lawyers to review this. Minor contract change requests are
+    done inhouse and cost nothing.</p>
```
Comment (brikis98) [inline pages/pricing/_other-services.html]: Although I like the sentiment, and we really don't hide any fees, we also don't display our enterprise pricing or consulting pricing on the website. Perhaps just keep it simple and say "Legal fee": the pricing on the page comes with our standard contract. If you request *significant* changes on that contract, we add a fee of XXX to pay our lawyers. Minor contract changes...

File: pages/pricing/_other-services.html [hunk 3]
```diff
@@ -13,5 +13,12 @@ <h2>Other Services</h2>
         </tr>
       </tbody>
     </table>
+
+    <h3>Miscellaneous Fees</h3>
+    <p><strong>Gruntwork has no hidden fees.</strong> The one fee we apply is a one-time charge of $4,000 for any significant requested
```
Comment (brikis98) [inline pages/pricing/_other-services.html]: I'd make it $5k based on our legal team's estimates.

**Comment by josh-padnick** [comment]
Made the requested changes. I think it sounds better. My one concern though is that how does someone distinguish from us simply passing around our costs versus us exorting a customer into accepting our "unreasonable" terms? For example, suppose we have an especially onerous clause or we offer no indemnity and then claim that a customer who wishes to challenge that will be charged $5k to do so.

I'll experiment with some language to address this point.

**Comment by brikis98** [comment]
What you have there LGTM. I'd add that this is a temporary stop gap. Soon, we'll have self-checkout with an online ToS. So the only people this will affect are those who refuse to sign up online.

**Comment by mcalhoun** [comment]
> Made the requested changes. I think it sounds better. My one concern though is that how does someone distinguish from us simply passing around our costs versus us exorting a customer into accepting our "unreasonable" terms? For example, suppose we have an especially onerous clause or we offer no indemnity and then claim that a customer who wishes to challenge that will be charged $5k to do so.

> I'll experiment with some language to address this point.

They always have the ability to negotiate and then not sign the deal, owing us nothing, right?

**Comment by josh-padnick** [comment]
Thx for the great feedback, guys. Two good points:
- With self-checkout, the customer can both see exactly what our contract is and understand clearly why we need to charge a fee to deviate from it.
- If the customer finds our terms onerous, they can always choose not to sign, though we'll of course make every effort to create a fair contract.

Merging and deploying now!

**Pull request merged**